bin/integration_test.sh

```diff
@@ -107,7 +107,7 @@ reportsuccess "THROTTLE=0 case submitted nothing!"
 # ==============================================================================
 newtest "THROTTLE=100 (submit everything) and make sure it prints 'accept'"
 
-OUTPUT=$(echo "${EVENT}" | THROTTLE=100 ./bin/run_invoke.sh 2>&1)
+OUTPUT=$(echo "${EVENT}" | THROTTLE=100 ./bin/run_invoke.sh 2>&1) || true
```

Comment:
I tweaked this one to be more like the other ones so when it fails, it's easier to debug.